app/static/js/int/main.js

```diff
@@ -0,0 +1,171 @@
+/*
+
+>> kasperkamperman.com - 2018-04-18
+>> https://www.kasperkamperman.com/blog/camera-template/
+
+*/
+
+
+video = document.getElementById('video');
+var takePhotoButton;
+
+var amountOfCameras = 0;
+var currentFacingMode = 'self';
+
+error_message = document.querySelector('#error-message');
+
+function displayErrorMessage(error_msg, error) {
+  error = error || "";
+  if (error) {
+    console.log(error);
+  }
+  error_message.innerText = error_msg;
+  error_message.classList.add("visible");
+}
+
+document.addEventListener('DOMContentLoaded', function (event) {
+  // do some WebRTC checks before creating the interface
+  DetectRTC.load(function () {
+    // do some checks
+    if (DetectRTC.isWebRTCSupported == false) {
+      alert(
+        '111 Please use Chrome, Firefox, iOS 11, Android 5 or higher, Safari 11 or higher',
+      );
+      displayErrorMessage(' 1111  use Chrome, Firefox, iOS 11, Android 5 or higher, Safari 11 or higher')
+    } else {
+      if (DetectRTC.hasWebcam == false) {
+        alert(' 2222 Please install an external webcam device. NoNoNoo');
+        displayErrorMessage("2222 Please install an external webcam device.")
```

Comment:
What do the numbers in front of the strings mean? If they don't mean anything we should remove them.